### DISCUSSION TITLE
ETA on the code release.

### DISCUSSION THREAD

**Discussion by JiamingSuen** [zju3dv/NeuralRecon#2] [General]
We plan to release the code within a month, stay tuned.

**Comment by ZX-Yin** [comment]
Nice work! Looking forward for the code. Thank you.

**Reply by nongmo677** [reply]
牛啊牛啊，期待大佬们，学习学习

**Comment by JiaqiuZhou** [comment]
Thank you for your excellent work! I want to start a 3D reconstruction work based on it. Waiting for the code release.

**Comment by heiwang1997** [comment]
Thanks for this great work!

**Comment by apoorvkh** [comment]
+1 on code release -- would be very helpful, thanks!

**Comment by Xin-Li24** [comment]
十分期待

**Comment by JustinMeans** [comment]
Impressive work, hope to see the code soon!

**Comment by tinuxnet** [comment]
Kudos!
+1 for the code ;)

Thank you

**Comment by meltoner** [comment]
would love to test this in ajetson nano

**Reply by JiamingSuen** [reply]
Thanks for your interest! Unfortunately, it would need a lot of work to deploy NeuralRecon to an ARM-based platform. You will have to train a much smaller model in order to run it in real-time on a Jetson GPU. You are more than welcome to try and we can discuss further.

**Reply by meltoner** [reply]
well thank you for your reply and supportive comment ! I will study your work further and attempt the next days. Currently, i am sorting out the mechanical and electrical parts of a small vehicle !

**Comment by apoorvkh** [comment]
In case others have not seen, the author added some of the code yesterday. Thank you!

**Comment by JiamingSuen** [comment]
We are happy to announce that the training and inference code of NeuralRecon is now released! We also included a real-time demo of NeuralRecon that can reconstruct custom videos captured by yourself, welcome to check it out!
